.jenkins

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent {
+        label 'DoNotRunAnythingOnMaster'
+    }
+    stages {
+        stage("Deploy documentation") {
```

Comment:
I would like to migrate the other builds to this jenkinsfile.  I plan on doing that later if you agree that it is better this way.

Reply:
Are we rebuilding the images on every pull request? Otherwise I'm OK with the move.

Reply:
Are we rebuilding the images on every pull request? Otherwise I'm OK with the move.

Reply:
Images will be cached and rebuilt if something changed.  That's how we do it on ArborX and Kokkos now.